packages/design-system/src/components/N8nMarkdown/Markdown.vue

```diff
@@ -129,7 +130,10 @@ export default {
 							const id = value.split('fileId:')[1];
 							return `src=${xss.friendlyAttrValue(imageUrls[id])}` || '';
 						}
-						if (!value.startsWith('https://')) {
+						// Only allow http requests to supported image files from the `static` directory
+						const isImageFile = value.split('#')[0].match(/\.(jpeg|jpg|gif|png|webp)$/) !== null;
```

Comment:
reminder to double check on these extensions with Max

Reply:
Checked. Added more extensions based on his feedback.